tests/lac/block_linear_operator_03.cc

```diff
@@ -41,8 +41,10 @@ int main()
 
   auto op_a = LinearOperator<>();
 
+
   {
-    std::array<std::array<decltype(op_a), 2>, 2> temp {op_a, op_a, op_a, op_a};
+    std::array<decltype(op_a), 2> a {{op_a, op_a}};
+    std::array<std::array<decltype(op_a), 2>, 2> temp {{a,a}};
```

Comment:
I suppose you could have done this using
```
  std::array<std::array<decltype(op_a), 2>, 2> temp {{op_a, op_a}, {op_a, op_a}};
```
Note the extra set of braces. Or does that not work either?